app/build.gradle

```diff
@@ -34,3 +34,9 @@ tasks.named('test', Test) {
         events "passed"
     }
 }
+
+tasks.withType(JavaCompile).configureEach {
```

Comment:
You can put this in the root build.gradle, which would mean you don't have to configure this in every module

Reply:
fixed. thanks for the catch